build-in-container/create-build-container.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+set -e
+
+docker build -t 'msft/mariner-container-build:2.0' .
```

Comment:
This should have a fully qualified path even when local (`mcr.microsoft.com/...`) since docker is a bit silly and if you don't have a local copy of a tag it will go and try to pull one from online sources (ie docker hub).

Reply:
Changed to mcr.microsoft.com/mariner-container-build:2.0